bgpd/bgp_vty.c

```diff
@@ -17167,6 +17190,16 @@ int bgp_config_write(struct vty *vty)
 					? ""
 					: "no ");
 
+		/* Send Hard Reset CEASE Notification for 'Administrative Reset'
+		 */
+		if (!!CHECK_FLAG(bgp->flags, BGP_FLAG_HARD_ADMIN_RESET) !=
+		    SAVE_BGP_HARD_ADMIN_RESET)
+			vty_out(vty, " %sbgp hard-administrative-reset\n",
+				CHECK_FLAG(bgp->flags,
+					   BGP_FLAG_HARD_ADMIN_RESET)
+					? ""
+					: "no ");
```

Comment:
We either need to add only the 'no bgp ...' statement or only the 'bgp ...' statement to 'sh running-config', depending on the default value of the config. Why add both explicitly?